### PR TITLE
Ci: Use hello-reason config

### PR DESCRIPTION
**Pull request by tatchi**
I copied the config from `hello-reason` repo since this one is apparently more up to date.

First commit is the raw copy/paste from `hello-reason` config. Second commit contains a few changes I've made.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,13 +63,13 @@
     "@reason-native/pastel": "*",
     "@reason-native/rely": "*",
     "@esy-ocaml/reason": "*",
-    "ocaml": "~4.7.0"
+    "ocaml": "~4.7.0",
```
Comment (tatchi) [inline package.json]: What about bumping Ocaml version to 4.8 ?

Comment (prometheansacrifice) [reply package.json]: Yes, we could do that. Perhaps match hello-reason in this regard

Comment (tatchi) [reply package.json]: Done :)

**Comment by prometheansacrifice** [comment]
Thanks @tatchi

**Pull request merged**